### ISSUE TITLE
Support Android Gradle Plugin 3.6?

### ISSUE THREAD

**Issue by Tolriq** [robolectric/robolectric#5299]
### Description
Lot's of tests no more works after updating AGP to 3.6 beta 1.

It seems they removed ```android.enableUnitTestBinaryResources=true``` and that generate the following stackTrace

```
java.lang.SecurityException: SHA-256 digest error for org/bouncycastle/jce/provider/BouncyCastleProvider.class
	at sun.security.util.ManifestEntryVerifier.verify(ManifestEntryVerifier.java:223)
	at java.util.jar.JarVerifier.processEntry(JarVerifier.java:243)
	at java.util.jar.JarVerifier.update(JarVerifier.java:230)
	at java.util.jar.JarVerifier$VerifierStream.read(JarVerifier.java:484)
	at java.io.FilterInputStream.read(FilterInputStream.java:133)
	at java.io.FilterInputStream.read(FilterInputStream.java:107)
	at org.robolectric.util.Util.copy(Util.java:46)
	at org.robolectric.util.Util.readBytes(Util.java:63)
	at org.robolectric.internal.bytecode.SandboxClassLoader.getByteCode(SandboxClassLoader.java:167)
	at org.robolectric.internal.bytecode.SandboxClassLoader.maybeInstrumentClass(SandboxClassLoader.java:119)
	at org.robolectric.internal.bytecode.SandboxClassLoader.lambda$findClass$0(SandboxClassLoader.java:112)
	at org.robolectric.util.PerfStatsCollector.measure(PerfStatsCollector.java:53)
	at org.robolectric.internal.bytecode.SandboxClassLoader.findClass(SandboxClassLoader.java:111)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at java.lang.Class.getDeclaredConstructors0(Native Method)
	at java.lang.Class.privateGetDeclaredConstructors(Class.java:2671)
	at java.lang.Class.getConstructors(Class.java:1651)
	at org.robolectric.util.inject.Injector.findConstructor(Injector.java:249)
	at org.robolectric.util.inject.Injector.inject(Injector.java:232)
	at org.robolectric.util.inject.Injector.lambda$new$0(Injector.java:115)
	at org.robolectric.util.inject.Injector$MemoizingProvider.get(Injector.java:485)
	at org.robolectric.util.inject.Injector.getInstanceInternal(Injector.java:213)
	at org.robolectric.util.inject.Injector.getInstance(Injector.java:197)
	at org.robolectric.util.inject.Injector.getInstance(Injector.java:191)
	at org.robolectric.internal.AndroidSandbox.lambda$new$0(AndroidSandbox.java:51)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```

### Steps to Reproduce
Use a test with for example:
```
    @Before
    fun prepare() {
        with(Shadows.shadowOf(MimeTypeMap.getSingleton())) {
            addExtensionMimeTypMapping("mp3", "audio/mp3")
            addExtensionMimeTypMapping("wav", "audio/wav")
            addExtensionMimeTypMapping("m3u8", "application/x-mpegurl")
        }
    }
```

### Robolectric & Android Version
RoboElectric 4.3 & 4.3.1, Test ran with SDK 21 to 28

### Link to a public git repo demonstrating the problem:
Can try to create one if necessary and the issue is not obvious.

**Comment by katajona** [comment]
I have the exact same problem.

**Comment by yshrsmz** [comment]
Related issue from google's issue tracker: https://issuetracker.google.com/issues/142580430

As suggested in the issue, adding a line below in `gradle.propeties` solved the problem for me.

```
android.jetifier.blacklist=.*bcprov.*
```

**Comment by polson** [comment]
I just spent way too much time on this thanks!

**Comment by fobo66** [comment]
Same error happens with AGP 4.0.0-alpha02.

**Comment by lewisevans** [comment]
Fixed it for me, Thanks!

**Comment by ansman** [comment]
This has been fixed as of jettifier `1.0.0-beta08` (released in `3.6.0-rc01` and `4.0.0-alpha06`) and this issue can probably be closed.

**Comment by hoisie** [comment]
Thanks for the details @ansman. Closing this issue since it seems to have been fixed in the updated gradle versions.
